### PR TITLE
Remove unneeded phpdocs

### PR DESCRIPTION
**Pull request by fabpot**
| Q             | A
| ------------- | ---
| Branch?       | 4.4
| Bug fix?      | no
| New feature?  | no 
| BC breaks?    | no     
| Deprecations? | no 
| Tests pass?   | yes    
| Fixed tickets | n/a
| License       | MIT
| Doc PR        | n/a

### CODE REVIEW

**Review by lyrixx — APPROVED** [review]

File: src/Symfony/Component/HttpKernel/Fragment/HIncludeFragmentRenderer.php [hunk 1]
```diff
@@ -34,7 +34,6 @@ class HIncludeFragmentRenderer extends RoutableFragmentRenderer
 
     /**
      * @param EngineInterface|Environment $templating            An EngineInterface or a Twig instance
-     * @param UriSigner                   $signer                A UriSigner instance
      * @param string                      $globalDefaultTemplate The global default content (it can be a template name or the content)
      * @param string                      $charset
```
Comment (chalasr) [inline src/Symfony/Component/HttpKernel/Fragment/HIncludeFragmentRenderer.php]: useless

**Review by chalasr — APPROVED** [review]

File: src/Symfony/Component/Form/Extension/Csrf/Type/FormTypeCsrfExtension.php [hunk 2]
```diff
@@ -54,8 +54,7 @@ public function __construct(CsrfTokenManagerInterface $defaultTokenManager, bool
     /**
      * Adds a CSRF field to the form when the CSRF protection is enabled.
      *
-     * @param FormBuilderInterface $builder The form builder
-     * @param array                $options The options
+     * @param array $options The options
```
Comment (Tobion) [inline src/Symfony/Component/Form/Extension/Csrf/Type/FormTypeCsrfExtension.php]: there are alot of useless `array $options` phpdocs as we already have the array typehint

**Comment by nicolas-grekas** [comment]
Thank you @fabpot.

File: src/Symfony/Component/Form/Extension/Csrf/Type/FormTypeCsrfExtension.php (hunk 2 above)
Comment (nicolas-grekas) [reply src/Symfony/Component/Form/Extension/Csrf/Type/FormTypeCsrfExtension.php]: removed while merging

File: src/Symfony/Component/CssSelector/Node/FunctionNode.php [hunk 3]
```diff
@@ -30,9 +30,8 @@ class FunctionNode extends AbstractNode
     private $arguments;
 
     /**
-     * @param NodeInterface $selector
-     * @param string        $name
-     * @param Token[]       $arguments
+     * @param string  $name
```
Comment (kaznovac) [inline src/Symfony/Component/CssSelector/Node/FunctionNode.php]: no additional value

File: src/Symfony/Component/Form/Extension/Csrf/CsrfExtension.php [hunk 4]
```diff
@@ -28,9 +28,8 @@ class CsrfExtension extends AbstractExtension
     private $translationDomain;
 
     /**
-     * @param CsrfTokenManagerInterface $tokenManager      The CSRF token manager
-     * @param TranslatorInterface|null  $translator        The translator for translating error messages
-     * @param string|null               $translationDomain The translation domain for translating
+     * @param TranslatorInterface|null $translator        The translator for translating error messages
+     * @param string|null              $translationDomain The translation domain for translating
```
Comment (kaznovac) [inline src/Symfony/Component/Form/Extension/Csrf/CsrfExtension.php]: inferable from context

File: src/Symfony/Component/Form/Extension/DependencyInjection/DependencyInjectionExtension.php [hunk 5]
```diff
@@ -25,9 +25,8 @@ class DependencyInjectionExtension implements FormExtensionInterface
     private $guesserServices;
 
     /**
-     * @param ContainerInterface $typeContainer
-     * @param iterable[]         $typeExtensionServices
-     * @param iterable           $guesserServices
+     * @param iterable[] $typeExtensionServices
+     * @param iterable   $guesserServices
```
Comment (kaznovac) [inline src/Symfony/Component/Form/Extension/DependencyInjection/DependencyInjectionExtension.php]: no additional value

File: src/Symfony/Component/Form/FormEvent.php [hunk 6]
```diff
@@ -22,8 +22,7 @@ class FormEvent extends Event
     protected $data;
 
     /**
-     * @param FormInterface $form The associated form
-     * @param mixed         $data The data
+     * @param mixed $data The data
```
Comment (kaznovac) [inline src/Symfony/Component/Form/FormEvent.php]: no additional value

File: src/Symfony/Component/Form/FormFactoryInterface.php [hunk 7]
```diff
@@ -23,9 +23,8 @@ interface FormFactoryInterface
      *
      * @see createBuilder()
      *
-     * @param string $type    The type of the form
-     * @param mixed  $data    The initial data
-     * @param array  $options The options
+     * @param string $type The type of the form
+     * @param mixed  $data The initial data
```
Comment (kaznovac) [inline src/Symfony/Component/Form/FormFactoryInterface.php]: kinda inferable (4 instances in this class)

File: src/Symfony/Component/Form/Tests/AbstractFormTest.php [hunk 8]
```diff
@@ -54,10 +54,9 @@ protected function tearDown()
     abstract protected function createForm();
 
     /**
-     * @param string                   $name
-     * @param EventDispatcherInterface $dispatcher
-     * @param string|null              $dataClass
-     * @param array                    $options
+     * @param string      $name
+     * @param string|null $dataClass
+     * @param array       $options
```
Comment (kaznovac) [inline src/Symfony/Component/Form/Tests/AbstractFormTest.php]: useless

File: src/Symfony/Component/HttpFoundation/Session/Storage/MockFileSessionStorage.php [hunk 9]
```diff
@@ -27,9 +27,8 @@ class MockFileSessionStorage extends MockArraySessionStorage
     private $savePath;
 
     /**
-     * @param string      $savePath Path of directory to save session files
-     * @param string      $name     Session name
-     * @param MetadataBag $metaBag  MetadataBag instance
+     * @param string $savePath Path of directory to save session files
+     * @param string $name     Session name
```
Comment (kaznovac) [inline src/Symfony/Component/HttpFoundation/Session/Storage/MockFileSessionStorage.php]: both kinda inferable

File: src/Symfony/Component/HttpFoundation/Session/Storage/NativeSessionStorage.php [hunk 10]
```diff
@@ -99,7 +99,6 @@ class NativeSessionStorage implements SessionStorageInterface
      *
      * @param array                         $options Session configuration options
```
Comment (kaznovac) [inline src/Symfony/Component/HttpFoundation/Session/Storage/NativeSessionStorage.php]: no additional value

File: src/Symfony/Component/HttpKernel/Event/KernelEvent.php [hunk 11]
```diff
@@ -27,10 +27,8 @@ class KernelEvent extends Event
     private $requestType;
 
     /**
-     * @param HttpKernelInterface $kernel      The kernel in which this event was thrown
-     * @param Request             $request     The request the kernel is currently processing
-     * @param int                 $requestType The request type the kernel is currently processing; one of
-     *                                         HttpKernelInterface::MASTER_REQUEST or HttpKernelInterface::SUB_REQUEST
+     * @param int $requestType The request type the kernel is currently processing; one of
```
Comment (kaznovac) [inline src/Symfony/Component/HttpKernel/Event/KernelEvent.php]: int|null

File: src/Symfony/Component/Routing/Loader/XmlFileLoader.php [hunk 12]
```diff
@@ -63,10 +63,9 @@ public function load($file, $type = null)
     /**
      * Parses a node from a loaded XML file.
      *
-     * @param RouteCollection $collection Collection to associate with the node
-     * @param \DOMElement     $node       Element to parse
-     * @param string          $path       Full path of the XML file being processed
-     * @param string          $file       Loaded file name
+     * @param \DOMElement $node Element to parse
```
Comment (kaznovac) [inline src/Symfony/Component/Routing/Loader/XmlFileLoader.php]: inferable

File: src/Symfony/Component/Translation/Dumper/DumperInterface.php [hunk 13]
```diff
@@ -24,8 +24,7 @@ interface DumperInterface
     /**
      * Dumps the message catalogue.
      *
-     * @param MessageCatalogue $messages The message catalogue
-     * @param array            $options  Options that are used by the dumper
+     * @param array $options Options that are used by the dumper
```
Comment (kaznovac) [inline src/Symfony/Component/Translation/Dumper/DumperInterface.php]: no additional value

File: src/Symfony/Component/Validator/ConstraintViolation.php [hunk 14]
```diff
@@ -32,22 +32,20 @@ class ConstraintViolation implements ConstraintViolationInterface
...
-     * @param mixed           $invalidValue    The invalid value that caused this
-     *                                         violation
-     * @param int|null        $plural          The number for determining the plural
-     *                                         form when translating the message
-     * @param string|null     $code            The error code of the violation
-     * @param Constraint|null $constraint      The constraint whose validation
-     *                                         caused the violation
-     * @param mixed           $cause           The cause of the violation
+     * @param string      $message         The violation message
+     * @param string      $messageTemplate The raw violation message
```
Comment (kaznovac) [inline src/Symfony/Component/Validator/ConstraintViolation.php]: lots of |null types can be stated in doc ($message, $messageTemplate, $propertyPath)

**Review by kaznovac — COMMENTED** [review]
some impovements

**Pull request merged**